auth-web/src/components/auth/manage-business/BusinessLookup.vue

```diff
@@ -170,9 +170,12 @@ export default defineComponent({
       if (!states.searchField || states.searchField?.length <= 2) {
         return
       }
+
       states.state = States.SEARCHING
       const searchStatus = null // search all (ACTIVE + HISTORICAL)
-      const legalType = launchdarklyServices.getFlag(LDFlags.AllowableBusinessSearchTypes)
```

Comment:
Previously, if LaunchDarkly was not available (or wrong key), this would return `null`.

Then, the Business Lookup Service would fallback to the complete list of entity types. This is not a good fallback.

The new fallback is "same as in Prod".

Reply:
I think you could also use launchdarklyServices.getFlag(LDFlags.AllowableBusinessSearchTypes, 'SP,GP...')

Reply:
I've changed that function recently so the fall backs should work, I think I used ublock origins to test it

Reply:
Good call. I've updated and tested this.